app/controllers/admin/messages/list.js

```diff
@@ -52,7 +52,10 @@ get columns() {
       valuePath       : 'sentAt',
       headerComponent : 'tables/headers/sort',
       isSortable      : true,
-      cellComponent   : 'ui-table/cell/cell-simple-date'
+      cellComponent   : 'ui-table/cell/cell-simple-date',
+      options         : {
+        dateFormat: 'D MMM, YYYY h:mm A'
```

Comment:
Isn't it same as date-time-short, then why override it with an already default option?